task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -67,18 +67,28 @@ spec:
     - name: test-format-result
       image: quay.io/redhat-appstudio/hacbs-test:latest
       script: |
-        ERR_MSG="$(echo -n $(cat $(workspaces.sanity-ws.path)/stderr.txt))"
-        ERR_MSG="${ERR_MSG:-unknown}"
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --arg MSG "${ERR_MSG: 0: 3000}" --null-input \
-          '{result: "ERROR", timestamp: $date, failures: $MSG}')
+        if [ ! -z $(cat $(workspaces.sanity-ws.path)/stderr.txt) ]
+        then
+          echo "The deprecated-image-check test meets the following error:"
+          cat $(workspaces.sanity-ws.path)/stderr.txt
+        fi
+
+        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input '{result: "ERROR", timestamp: $date}')
         HACBS_TEST_OUTPUT=$(jq -rce --arg date $(date +%s) \
           '.[] | { result: (if (.failures | length > 0) then "FAILURE" else "SUCCESS" end),
                    timestamp: $date,
                    namespace,
                    successes,
-                   failures: (.failures // [])|map(.metadata.details.name) | unique
+                   failures: (.failures | length)
                  }' $(workspaces.sanity-ws.path)/deprecated_image_check_output.json || true)
 
+        # Log out the failing runs
+        if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+        then
+          echo "The deprecated-image-check test fails with the following runs:"
+          echo $(jq '.[].failures' $(workspaces.sanity-ws.path)/deprecated_image_check_output.json)
```

Comment:
Maybe we can use the .[].failures // []|map(.metadata.details.name) jq filter here to display only the error names.